tools/clang/lib/SPIRV/SpirvModule.cpp

```diff
@@ -34,7 +34,7 @@ SpirvModule::~SpirvModule() {
     exec->releaseMemory();
   for (auto *str : constStrings)
     str->releaseMemory();
-  for (auto *d : debugSources)
+  for (auto *d : sources)
```

Comment:
you also have to add a new for loop that iterates over `debugInstructions` that has been added to SpirvModule to release their memory.

Reply:
done